earn/src/components/boost/ImportBoostWidget.tsx

```diff
@@ -275,6 +277,16 @@ export default function ImportBoostWidget(props: ImportBoostWidgetProps) {
   const modifyData = useMemo(() => {
     if (!cardInfo) return undefined;
     const { position } = cardInfo;
+    const maxSlippage = parseFloat(maxSlippagePercentage) / 100;
+    const maxBorrowAmount0 = borrowAmount0
+      .recklessMul(1 + maxSlippage)
+      .toBigNumber()
+      .add(2);
+    const maxBorrowAmount1 = borrowAmount1
+      .recklessMul(1 + maxSlippage)
+      .toBigNumber()
+      .add(2);
+    const combinedMaxBorrowAmount = maxBorrowAmount0.add(maxBorrowAmount1.mul(GN.Q(112).toBigNumber()));
```

Comment:
If `BigNumber` has a `shl` function, I would shift left 112 instead of converting Q112 to a BigNumber and multiplying.

Reply:
Looks like `shiftedBy` is a thing

Reply:
Wrong BigNumber library, but ethers Bignumber does have `shl`.